client/src/app/editor/file-outputs/file-outputs.component.html

```diff
@@ -60,5 +59,11 @@ <h4 class="ml-no-information-message-headline">No outputs available</h4>
         <p class="ml-no-information-message-text">Select an execution that has generated output files.</p>
       </div>
     </ng-template>
+    <ng-template #loadingSpinner>
+      <div class="spinner-wrapper">
+        <mat-progress-spinner mode="indeterminate"></mat-progress-spinner>
+        <span class="no-output-text">Loading outputs</span>
```

Comment:
Oh and while we're at it... let's please get rid off this span here 😅

Thanks!